recipes-kernel/linux/linux-toradex-kmeta.inc

```diff
@@ -0,0 +1,14 @@
+SRCREV_meta = "c6baa89ab5b0a94d177a098f01e3d5e79d57fdf7"
+SRCREV_meta_use-head-next = "${AUTOREV}"
+
+KMETABRANCH = "toradex_5.4.y"
+KMETA = "kernel-meta"
+
+KMACHINE_apalis-imx8x-v11a = "apalis-imx8x"
+KMACHINE_colibri-imx7-emmc = "colibri-imx7"
+KMACHINE_colibri-imx8x-v10b = "colibri-imx8x"
```

Comment:
Whats up with that machines? Are they needed in our scope?

Reply:
They are not needed in our scope. But meta-ci.os may be used by anyone, even by users with those machines. I would leave it in.

Reply:
Ok